### PR TITLE
Map randomized int block state providers

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

File: mappings/net/minecraft/world/gen/stateprovider/RandomizedIntBlockStateProvider.mapping [hunk 1]
```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_5929 net/minecraft/world/gen/stateprovider/RandomizedIntBlockStateProvider
+	COMMENT A {@linkplain BlockStateProvider block state provider} that randomizes a single {@link net.minecraft.state.property.IntProperty} of a block state provided by another provider.
```
Comment (liach) [inline mappings/net/minecraft/world/gen/stateprovider/RandomizedIntBlockStateProvider.mapping]:
```suggestion
	COMMENT A {@linkplain BlockStateProvider block state provider} that randomizes a single {@link IntProperty} of a block state provided by another provider.
```
no need for qualification as it's already used in fields

File: mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping [hunk 2]
```diff
@@ -7,6 +7,8 @@ CLASS net/minecraft/class_5464 net/minecraft/world/gen/feature/ConfiguredFeature
 	FIELD field_29088 LAPIS_ORE_TARGETS Lcom/google/common/collect/ImmutableList;
 	FIELD field_29090 IRON_CONFIG Lnet/minecraft/class_3124;
 	FIELD field_29091 REDSTONE_CONFIG Lnet/minecraft/class_3124;
+	FIELD field_29511 RANDOM_BERRIES_CAVE_VINES_BODY_PROVIDER Lnet/minecraft/class_4657;
+	FIELD field_29512 RANDOM_AGE_CAVE_VINES_HEAD_PROVIDER Lnet/minecraft/class_5929;
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping:12]: Can the names be more concise?

Comment (haykam821) [reply mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping:12]: Should the `RANDOM_X_` prefix be removed? I don't think any other parts of the name could be removed without losing information.

File: mappings/net/minecraft/world/gen/stateprovider/RandomizedIntBlockStateProvider.mapping (hunk 1 above)
Comment (haykam821) [reply mappings/net/minecraft/world/gen/stateprovider/RandomizedIntBlockStateProvider.mapping]: Fixed in 12bc1e25c87248206e10b07df41b2576097cd52d

File: mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/world/gen/feature/ConfiguredFeatures.mapping:12]: The random is needed info still. This is fine

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Comment by liach** [comment]
@haykam821 mind fix this merge conflict real quick?

**Review by enbrain — APPROVED** [review]

**Pull request merged**